Dockerfile

```diff
@@ -1,35 +1,21 @@
-ARG RUST_VERSION=1.82.0
-ARG APP_NAME=gateway
-FROM rust:${RUST_VERSION}-slim-bookworm AS build
-ARG APP_NAME
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
```

Comment:
there isn't more conventional image for this chef thingy?

Reply:
also, use at least a major version notion instead of going fully latest. to have a semi reproducible builds